cogeo_mosaic/backends/base.py

```diff
@@ -196,6 +223,11 @@ def update(
         **kwargs,
     ):
         """Update existing MosaicJSON on backend."""
+        if self.mode == "r":
```

Comment:
`if 'w' not in self.mode`? It would be good to look up with the valid modes are to `open`.